src/layout/components/GridContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+/* Grid container for the main page */
+export const GridContainer = styled('div')`
+    max-width: 960px;
+    width: 80%;
+    display: grid;
+    margin: auto; 
+    margin-top: 2em;
+    grid-template-columns: repeat(auto-fit, minmax(min(350px, 100%), 1fr));
+    grid-column-gap: 2em;
+    grid-row-gap: 2em;
```

Comment:
There is a style `grid-gap` which includes both `grid-column-gap` and `grid-row-gap`.

Reply:
Done :)